.github/workflows/ibmc-cluster-teardown.yml

```
@@ -0,0 +1,125 @@
+    inputs:
+      cluster_name:
+        description: 'Cluster name to tear down'
+        required: true
+        default: 'kubevirt-plugin-ci'
+        type: string
+  workflow_call:
+    inputs:
+      cluster_name:
+        description: 'Cluster name to tear down'
+        required: true
+        default: 'kubevirt-plugin-ci'
```

actionlint (1.7.12)

[error] input "cluster_name" of workflow_call event has the default value "kubevirt-plugin-ci", but it is also required. if an input is marked as required, its default value will never be used

(events)